src/pages/IstioRuleDetails/IstioRuleInfo.tsx

```diff
@@ -72,19 +72,62 @@ class IstioRuleInfo extends React.Component<RuleDetailsId, RuleInfoState> {
       });
   }
 
+  // Handlers and Instances have a type attached to the name with '.'
+  // i.e. handler=myhandler.kubernetes
+  validateParams(parsed: ParsedSearch): boolean {
+    if (!parsed.type || !parsed.name || this.state.actions.length === 0) {
+      return false;
+    }
+    let validationType = ['handler', 'instance'];
+    if (parsed.type && validationType.indexOf(parsed.type) < 0) {
+      return false;
+    }
+    for (let i = 0; i < this.state.actions.length; i++) {
+      let splitName = parsed.name.split('.');
+      if (splitName.length !== 2) {
+        continue;
+      }
```

Comment:
...How is this working?  Don't you need to set else {break;} ? we

Reply:
Good catch, yes, the split should be out of the loop and break if it fails, not iterate on all the actions.

Reply:
@aljesusg fixed

Reply:
Waiting for Travis to merge @lucasponce